### PR TITLE
feat(console): sign in experience welcome page

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add welcome page for sign in experience.

TODO: LOG-1747 show a guide on clicking "get started".

## Linear Issue Reference

LOG-1745

## Testing

截屏2022-05-06 下午5 58 20

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by demonzoo — APPROVED** [review]

File: packages/console/src/pages/SignInExperience/index.tsx [hunk 1]
```diff
@@ -54,6 +56,18 @@ const SignInExperience = () => {
     toast.success(t('application_details.save_success'));
   });
 
+  if (!settings && !settingsError) {
+    return <div>loading</div>;
+  }
+
+  if (settingsError) {
+    return <div>{settingsError.body.message}</div>;
```
Comment (xiaoyijun) [inline packages/console/src/pages/SignInExperience/index.tsx:64]: Will it be handled by the default error page?

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/console/src/pages/SignInExperience/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/SignInExperience/index.tsx:64]: Not sure about that, align them later.

**Pull request merged**